tests/java_integers.rs

```diff
@@ -0,0 +1,81 @@
+extern crate error_chain;
+extern crate jni;
+
+use error_chain::ChainedError;
+use jni::{InitArgsBuilder, JavaVM, JNIEnv};
+use jni::objects::JObject;
+use jni::objects::JValue;
+use jni::sys::JNI_VERSION_1_8;
+
+fn print_exception(env: &JNIEnv) {
+    let exception_occurred = env.exception_check().unwrap_or_else(
+        |e| panic!(format!("{:?}", e)),
+    );
+    if exception_occurred {
+        env.exception_describe().unwrap_or_else(
+            |e| panic!(format!("{:?}", e)),
+        );
+    }
+}
+
+#[test]
+fn test_java_integers() {
+    unsafe {
+        let jvm_args = InitArgsBuilder::new()
+            .version(JNI_VERSION_1_8)
+            .option("-Xcheck:jni")
+            .option("-Xdebug")
+            .build()
+            .unwrap_or_else(|e| {
+                panic!(format!("{}", e.display_chain().to_string()));
+            });
+
+        let (jvm, env) = JavaVM::new(jvm_args).unwrap_or_else(|e| {
+            panic!(format!("{}", e.display_chain().to_string()));
+        });
+
+        for int_value in -10..10 {
+            for array_length in 1..50 {
+                env.push_local_frame(0).unwrap_or_else(|e| {
```

Comment:
Why a local frame of zero capacity? The docs don't seem to say anything regarding a default capacity: https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#PushLocalFrame